site/docs/relations/physical_relations.md

```diff
@@ -71,6 +71,29 @@ The merge equijoin does a join by taking advantage of two sets that are sorted o
 | Post Join Predicate | An additional expression that can be used to reduce the output of the join operation post the equality condition. Minimizes the overhead of secondary join conditions that cannot be evaluated using the equijoin keys. | Optional, defaults true.    |
 | Join Type           | One of the join types defined in the Join operator.                                                                                                                                                                     | Required                    |
 
+
+
+## Mark Join Operator
+
+A mark join internally scans the left side, constructing a hash table that is used to mark the right side as having a join partner on the left side.  This mark can end up being True, False, or NULL.  The NULL mark is used to indicate that the right side does not have a join partner on the left side.  The mark join operator is used to implement semi-joins, anti-joins, and other join types that are not equijoins.
```

Comment:
Given the mark might be null then it sounds like this is a right join.  In other words, there is one output row for each row on the right side.  Is that true?  Can we describe that more clearly?

Reply:
A regular right/outer join can still emit multiple output rows per input row, a mark join preserves the cardinality of the left hand side exactly.